src/structures/GuildChannel.js

```diff
@@ -514,7 +514,17 @@ class GuildChannel extends Channel {
   get deletable() {
     return this.permissionsFor(this.client.user).has(Permissions.FLAGS.MANAGE_CHANNELS, false);
   }
-
+  
+  /**
+   * Whether the channel is readable by the client user
+   * @type {boolean}
+   * @readonly
+   */
+  
```

Comment:
This line needs to be removed so the docs touch the getter. Otherwise, the docs parser will not associate the docs block with the readable property.